service_test.go

```diff
@@ -3,6 +3,9 @@ package micro
 import (
 	"context"
 	"errors"
+	"go-micro.dev/v4/server"
```

Comment:
external imports should be separated from stdlib, please see other files as an example

Reply:
I'm sorry it was my oversight.